tests/boards/altera_max10/qspi/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  test:
+    platform_whitelist: altera_max10
```

Comment:
Will this work on any MAX10 device, are there any pre-conditions on its setup?

Reply:
This should work on any max10 device provided it's softcpu has support for QSPI soft IP